cmd/access/node_builder/access_node_builder.go

```diff
@@ -591,9 +591,6 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 				return errors.New("execution-data-max-search-ahead must be greater than 0")
 			}
 		}
-		if builder.rpcConf.ConnectionPoolSize == 0 {
```

Comment:
can you add a note in the flag description about the default value and meaning of 0?